Makefile.toml

```diff
@@ -159,6 +159,11 @@ BUILDSYS_OVF_TEMPLATE = "${BUILDSYS_ROOT_DIR}/variants/${BUILDSYS_VARIANT}/templ
 # The default name of uploaded OVAs; override by setting VMWARE_VM_NAME
 VMWARE_VM_NAME_DEFAULT = "${BUILDSYS_NAME}-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-v${BUILDSYS_VERSION_IMAGE}-${BUILDSYS_VERSION_BUILD}"
 
+# Config file for Boot Configuration initrd generation
+BOOT_CONFIG_INPUT = "${BUILDSYS_ROOT_DIR}/bootconfig-input"
+# Boot Configuration initrd
+BOOT_CONFIG = "${BUILDSYS_ROOT_DIR}/bootconfig.data"
```

Comment:
I don't think it's possible to override these variables with `cargo make -e`. But maybe that's ok.

Reply:
Good point :thinking: 

IMHO, I think that's probably ok considering we want to name the initrd a specific way.  I suppose flexibility in the input file name would be nice, but I'm not sure how much it gets us.